static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -43,27 +46,40 @@ function SampleImages({groupId}: Props) {
     })
     .splice(0, 5);
 
+  const hasImages = filteredResources.length > 0;
+
+  let body: React.ReactNode;
```

Comment:
m: Can we extract this into it's own component?

Reply:
I extract everything in the body into its own component